// radar.proto
syntax = "proto3";

package radar;

message StreamRequest {
  int32 refresh_interval_ms = 1;
  string filter = 2; // varsa
}

message RadarTarget {
  // YENÄ° ALAN
  string id = 6;

  double lat = 1;
  double lon = 2;
  int32 velocity = 3;
  int32 baro_altitude = 4;
  int32 geo_altitude = 5;
}

service RadarService {
  rpc StreamRadarTargets (StreamRequest) returns (stream RadarTarget);
}
